render/detailed_node.go

```diff
@@ -151,6 +103,56 @@ func MakeDetailedNode(r report.Report, n RenderableNode) DetailedNode {
 	}
 }
 
+func addConnectionsTable(tables *tables, connections []Row, r report.Report, n RenderableNode) {
+	sec := r.Window.Seconds()
+	rate := func(u *uint64) (float64, bool) {
+		if u == nil {
+			return 0.0, false
+		}
+		if sec <= 0 {
+			return 0.0, true
+		}
+		return float64(*u) / sec, true
+	}
+	shortenByteRate := func(rate float64) (major, minor string) {
+		switch {
+		case rate > 1024*1024:
+			return fmt.Sprintf("%.2f", rate/1024/1024), "MBps"
+		case rate > 1024:
+			return fmt.Sprintf("%.1f", rate/1024), "KBps"
+		default:
+			return fmt.Sprintf("%.0f", rate), "Bps"
+		}
+	}
+
+	rows := []Row{}
+	if n.EdgeMetadata.MaxConnCountTCP != nil {
+		rows = append(rows, Row{"TCP connections", strconv.FormatUint(*n.EdgeMetadata.MaxConnCountTCP, 10), "", false})
```

Comment:
No I can't:

`../src/github.com/weaveworks/scope/render/detailed_node.go:130: too few values in struct initializer`